monitoring/user-workload-monitoring.adoc

```diff
@@ -0,0 +1,17 @@
+[id="user-workload-monitoring"]
+= User workload monitoring
+include::modules/common-attributes.adoc[]
+:context: user-workload-monitoring
+
+toc::[]
+
+You can use the Prometheus Cluster Monitoring stack to monitor your applications, services, or other custom workloads in addition to monitoring the cluster. This way, you do not need to use an additional monitoring solution. This helps keeping monitoring centralized.
+Additionally, you can extend the access to the custom metrics beyond cluster administrators. This enables developers and arbitrary users to access the metrics.
```

Comment:
Let's add a paragraph here that opting into user workload monitoring is mutually exclusive with either a) a custom installation of Prometheus Operator and b) installing Prometheus Operator via OLM.

cc @brancz